Formula/android-sdk.rb

```diff
@@ -117,6 +125,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert_match version.to_s, shell_output("#{prefix}/tools/emulator -version")
```

Comment:
`emulator -version` isn't a recognized command. I don't think it was working in 24 either though